node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,5 @@
+const fs = require('fs');
+const randomNumber = Math.random() + '' + '\n';
```

Comment:
You don't need that extra string in there: `const randomNumber = Math.random() + '\n';` would do the same thing.